DFC.ServiceTaxonomy.GraphSync/GraphSyncers/VisualiseGraphSyncer.cs

```diff
@@ -419,8 +419,7 @@ private static string GetContHasName(Dictionary<string, object> properties, stri
         {
             var detailCommand = new List<IQuery<object?>>
             {
-                new CosmosDbNodeAndNestedOutgoingRelationshipsQuery(
-                    $"select * from c where c.id in ({string.Join(',', ids)})|{contentType}")
+                new CosmosDbNodeAndNestedOutgoingRelationshipsQuery("SELECT * FROM c WHERE c.id in (@id0)", string.Join(',', ids), contentType)
```

Comment:
Is this right? looks a bit odd - i imaging @id0 gets expanded? is it just the name thats weird?

Reply:
Have added an additional parameter to support custom parameter names